certified-connectors/Iconfirm/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4257 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ICONFIRM",
```

Comment:
@iconfirm-development Can you please remove capitalization for your connector title to maintain consistency with other connectors? You can use camelcase for your connector like you have used in your website title like below:

`"title": "Iconfirm"`

![image](https://user-images.githubusercontent.com/56276394/156248648-72c55ed1-3e3d-44de-a398-c71c467919d7.png)

Reply:
Have updates the title now